tensorflow_datasets/structured/HTRU2.py

```diff
@@ -0,0 +1,129 @@
+    issn = {0035-8711},
+    doi = {10.1093/mnras/stw656},
+    url = {https://doi.org/10.1093/mnras/stw656},
+    eprint = {http://oup.prod.sis.lan/mnras/article-pdf/459/1/1104/8115310/stw656.pdf},
+}
+
+"""
+
+_DESCRIPTION = """\
+HTRU2 is a data set which describes a sample of pulsar candidates collected during the High Time Resolution Universe Survey (South).
+Pulsars are a rare type of Neutron star that produce radio emission detectable here on Earth. 
+They are of considerable scientific interest as probes of space-time, the inter-stellar medium, and states of matter.
+As pulsars rotate, their emission beam sweeps across the sky, and when this crosses our line of sight, produces a detectable pattern of broadband radio emission.
+As pulsars rotate rapidly, this pattern repeats periodically. 
+Thus, pulsar search involves looking for periodic radio signals with large radio telescopes.
+Each pulsar produces a slightly different emission pattern, which varies slightly with each rotation.
+Thus a potential signal detection known as a 'candidate', is averaged over many rotations of the pulsar, as determined by the length of an observation.
+In the absence of additional info, each candidate could potentially describe a real pulsar.
+However, in practice almost all detections are caused by radio frequency interference (RFI) and noise, making legitimate signals hard to find.
+Machine learning tools are now being used to automatically label pulsar candidates to facilitate rapid analysis. 
+Classification systems in particular are being widely adopted, which treat the candidate data sets as binary classification problems. 
+Here the legitimate pulsar examples are a minority positive class, and spurious examples the majority negative class. 
+At present multi-class labels are unavailable, given the costs associated with data annotation.
+The data set shared here contains 16,259 spurious examples caused by RFI/noise, and 1,639 real pulsar examples. 
+These examples have all been checked by human annotators.
+"""
+
+_URL = "http://archive.ics.uci.edu/ml/machine-learning-databases/00372/HTRU2.zip"
+
+
+class Htru2(tfds.core.GeneratorBasedBuilder):
+  """Dataset for Predicting a Pulsar Star"""
+
+  VERSION = tfds.core.Version('2.0.0',
+                              experiments={tfds.core.Experiment.S3: False})
+
+  def _info(self):
+    return tfds.core.DatasetInfo(
+        builder=self,
+        description=_DESCRIPTION,
+        features=tfds.features.FeaturesDict({
```

Comment:
It might be better to define all the features except "Class" in a FEATURES_DICT OrderedDict and then define your tfds.features.FeatureDict having "features" and "classes"
Refer titanic.py
https://github.com/tensorflow/datasets/blob/4adb5224f3d8dc3243252585a815c652f4f9bac5/tensorflow_datasets/structured/titanic.py